multi-agent/tests/unit/scripts/test_migrate_builtin_reverse_no_key.py

```
@@ -0,0 +1,57 @@
+"""Regression test: reverse migration must abort when overlays contain
+encrypted fields but no --encryption-key was provided."""
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from global_utils.utils.crypto import FERNET_PREFIX
+
+
+def _make_mock_db(overlay_fields: dict):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_make_mock_db`

(ANN202)